<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_perso</class>
 <widget class="QDialog" name="Dialog_perso">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_BGPersonnel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>700</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/Image/Images/BGSponsoring.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labe_TitlePerso">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>0</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>LEMON MILK</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(134, 47, 147,50);
border-bottom-left-radius: 10px;
border-bottom-right-radius: 10px;</string>
   </property>
   <property name="text">
    <string>GESTION DES PERSONNELS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="gB_infopersonnel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>501</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgba(199, 153, 207, 200);
background: rgba(199, 153, 207, 100); 
border-radius: 10px;
</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_DateDeNaissance">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(134, 47, 147, 50);
	border: 2px solid rgb(134, 47, 147);
		border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>Date de naissance</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_CIN">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgba(134, 47, 147, 50);
		border: 2px solid rgba(134, 47, 147, 200);
			border-radius: 10px;
}
QLineEdit:hover {
	background-color: rgba(134, 47, 147, 50);
		border: 2px solid rgba(47, 127, 147, 200);
			border-radius: 10px;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_InsererPhotoPerso">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>180</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgba(199, 153, 207, 200);
border-radius: 10px;
}
QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(117, 201, 187, 200), stop:1 rgba(199, 153, 207, 200));
}</string>
    </property>
    <property name="text">
     <string>Inserer photo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Prenom">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(134, 47, 147, 50);
	border: 2px solid rgb(134, 47, 147);
		border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>Prenom</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Nom">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(134, 47, 147, 50);
	border: 2px solid rgb(134, 47, 147);
		border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>Nom</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Adresse">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(134, 47, 147, 50);
	border: 2px solid rgb(134, 47, 147);
		border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>Adresse</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Nom">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgba(134, 47, 147, 50);
		border: 2px solid rgba(134, 47, 147, 200);
			border-radius: 10px;
}
QLineEdit:hover {
	background-color: rgba(134, 47, 147, 50);
		border: 2px solid rgba(47, 127, 147, 200);
			border-radius: 10px;
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Prenom">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgba(134, 47, 147, 50);
		border: 2px solid rgba(134, 47, 147, 200);
			border-radius: 10px;
}
QLineEdit:hover {
	background-color: rgba(134, 47, 147, 50);
		border: 2px solid rgba(47, 127, 147, 200);
			border-radius: 10px;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_CIN">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(134, 47, 147, 50);
	border: 2px solid rgb(134, 47, 147);
		border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>CIN</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Adresse">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgba(134, 47, 147, 50);
		border: 2px solid rgba(134, 47, 147, 200);
			border-radius: 10px;
}
QLineEdit:hover {
	background-color: rgba(134, 47, 147, 50);
		border: 2px solid rgba(47, 127, 147, 200);
			border-radius: 10px;
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Email">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgba(134, 47, 147, 50);
		border: 2px solid rgba(134, 47, 147, 200);
			border-radius: 10px;
}
QLineEdit:hover {
	background-color: rgba(134, 47, 147, 50);
		border: 2px solid rgba(47, 127, 147, 200);
			border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Email">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(134, 47, 147, 50);
	border: 2px solid rgb(134, 47, 147);
		border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_imageperso">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_DateDeNaissance">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QDateEdit
{
    subcontrol-origin: padding;
    subcontrol-position: top right;
    selection-background-color: #111;
    selection-color: yellow;
    color: white;
    background-color: rgba(134, 47, 147, 50);
    border-style: solid;
    border: 1px solid #862f93;
    border-radius: 5;
    padding: 1px 0px 1px 20px;
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="displayFormat">
     <string>dd-MM-yyyy</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pB_TirerParEmail">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>600</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(199, 153, 207, 200);
border-radius: 10px;
}
QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(117, 201, 187, 200), stop:1 rgba(199, 153, 207, 200));
}</string>
   </property>
   <property name="text">
    <string>Trier par Email</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_ModifierP">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>170</y>
     <width>110</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(199, 153, 207, 200);
border-radius: 10px;
}
QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(117, 201, 187, 200), stop:1 rgba(199, 153, 207, 200));
}</string>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_CIN">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>110</y>
     <width>130</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox
{
    subcontrol-origin: padding;
    subcontrol-position: top right;
    selection-background-color: #111;
    selection-color: yellow;
    color: white;
    background-color: rgba(134, 47, 147, 50);
    border-style: solid;
    border: 1px solid #1e1e1e;
    border-radius: 5;
    padding: 1px 0px 1px 20px;
}</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_Perso">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>180</y>
     <width>341</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(199, 153, 207, 100);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_SupprimerP">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>200</y>
     <width>110</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(199, 153, 207, 200);
border-radius: 10px;
}
QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(117, 201, 187, 200), stop:1 rgba(199, 153, 207, 200));
}</string>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_TireParCIN">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>600</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(199, 153, 207, 200);
border-radius: 10px;
}
QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(117, 201, 187, 200), stop:1 rgba(199, 153, 207, 200));
}</string>
   </property>
   <property name="text">
    <string>Trier par CIN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_PorteP">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>150</y>
     <width>110</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(199, 153, 207, 200);
border-radius: 10px;
}
QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(117, 201, 187, 200), stop:1 rgba(199, 153, 207, 200));
}</string>
   </property>
   <property name="text">
    <string>Porte</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SupprimerP">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>150</y>
     <width>110</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(199, 153, 207, 200);
border-radius: 10px;
}
QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(117, 201, 187, 200), stop:1 rgba(199, 153, 207, 200));
}</string>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_AjouterP">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>140</y>
     <width>110</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(199, 153, 207, 200);
border-radius: 10px;
}
QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(117, 201, 187, 200), stop:1 rgba(199, 153, 207, 200));
}</string>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_Personnel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>701</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(199, 153, 207, 100);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_TirerParNom">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>600</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(199, 153, 207, 200);
border-radius: 10px;
}
QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(117, 201, 187, 200), stop:1 rgba(199, 153, 207, 200));
}</string>
   </property>
   <property name="text">
    <string>Trier par Nom</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Fermer">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>650</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(199, 153, 207, 200);
border-radius: 15px;
}
QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(189, 96, 96, 200), stop:1 rgba(199, 153, 207, 200));
}</string>
   </property>
   <property name="text">
    <string>Fermer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Info">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>60</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(134, 47, 147, 25);
	border: 2px solid rgba(134, 47, 147, 100);
		border-radius: 10px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
